### ISSUE TITLE
When a database pod is restarted there it a temp created in the /pgdata directory that takes up a lot of space

### ISSUE THREAD

**Issue by alrooney** [CrunchyData/postgres-operator#1957]
pgo 4.5.0.  I noticed that when the database pod is restarted a cluster_temp directory is created in /pgdata e.g.
```
$ kk exec -ti retroelk-645fb8fdcd-49gvf -c database -n pgo -- bash
bash-4.2$ ls -l /pgdata
total 8
drwx--S--- 20 postgres postgres 4096 Oct 11 16:09 retroelk
drwx--S--- 21 postgres postgres 4096 Oct 11 15:08 retroelk_tmp
bash-4.2$ 
```
Is it safe to delete this directory?  What is its purpose?  It takes up a lot of space on the database pvc.

**Comment by jkatz** [comment]
This directory is created as part of initializing an existing cluster, based on the cluster initialization method you've chosen:

https://github.com/CrunchyData/crunchy-containers/blob/master/bin/postgres-ha/bootstrap/pre-bootstrap.sh#L455-L470

This should later be removed though:

https://github.com/CrunchyData/crunchy-containers/blob/master/bin/postgres-ha/bootstrap/create-from-existing.sh#L24

Long story short, it is safe to delete

@andrewlecuyer: Any thoughts on why this may not be remaining after the fact?
